src/lsp/cobol_cfg/cobol_cfg.ml

```diff
@@ -0,0 +1,15 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
```

Comment:
Something is wrong about this header. For some time I think we added the `2024`, didn't we?

Reply:
Ah! this part of the code is not AGPL. Please use the same header as in `cfg_jumps.ml` for every file in this library.